packages/atlas/src/components/TablePaymentsHistory/TablePaymentsHistory.tsx

```diff
@@ -21,54 +33,188 @@ const COLUMNS: TableProps['columns'] = [
   {
     Header: 'Date',
     accessor: 'date',
+    width: 100,
   },
   {
     Header: 'Type',
     accessor: 'type',
+    width: 100,
+  },
+  {
+    Header: 'Sender',
+    accessor: 'sender',
+    width: 100,
   },
   {
     Header: 'Amount',
     accessor: 'amount',
+    width: 100,
   },
   {
-    Header: 'Channel balance',
-    accessor: 'channelBalance',
+    Header: 'Description',
+    accessor: 'description',
+    width: 200,
   },
 ]
 
+const paymentTypeMappings: Record<
+  PaymentType,
+  {
+    title: string
+    icon?: ReactElement
+  }
+> = {
+  'nft-sale': {
+    title: 'NFT sale',
+    icon: <SvgActionNft />,
+  },
+  'nft-royalty': {
+    title: 'NFT royalty',
+    icon: <SvgActionCrown />,
+  },
+  'claimed-reward': {
+    title: 'Claimed reward',
+    icon: <SvgActionRevenueShare />,
+  },
+  'withdrawal': {
+    title: 'Withdrawal',
+    icon: <SvgActionRevenueShare />,
+  },
+  'ypp-reward': {
+    title: 'YPP reward',
+    icon: <SvgActionRevenueShare />,
+  },
+  'council-reward': {
+    title: 'Council reward',
+    icon: <SvgActionCouncil />,
+  },
+  'direct-payment': {
+    title: 'Direct payment',
+    icon: <SvgActionPayment />,
+  },
+  'revenue-share': {
+    title: 'Revenue share',
+    icon: <SvgActionRevenueShare />,
+  },
+}
+
+type PaymentType =
+  | 'nft-sale'
+  | 'nft-royalty'
+  | 'claimed-reward'
+  | 'withdrawal'
+  | 'ypp-reward'
+  | 'council-reward'
+  | 'direct-payment'
+  | 'revenue-share'
+
 const tableEmptyState = {
   title: 'No payments here yet',
   description:
     'Here you will see proceedings to your channel balance from sold NFTs and royalties, claimed council rewards, direct payments from members to your channel and withdrawals from channel balance.',
   icon: <SvgEmptyStateIllustration />,
 }
 
-type PaymentType = 'nft-sale' | 'nft-royalty' | 'claimed-reward' | 'withdrawal' | 'ypp-reward'
-
 export type PaymentHistory = {
   type: PaymentType
   block: number
   date: Date
-  channelBalance: BN
+  sender: string
   amount: BN
+  description?: string
 }
 
 export type TablePaymentsHistoryProps = {
   data: PaymentHistory[]
 }
 
 export const TablePaymentsHistory: FC<TablePaymentsHistoryProps> = ({ data }) => {
+  const [dialogText, setDialogText] = useState('')
   const mappedData: TableProps['data'] = useMemo(
     () =>
       data.map((data) => ({
         date: <Date date={data.date} />,
         type: <Type type={data.type} />,
         amount: <TokenAmount tokenAmount={data.amount} />,
```

Comment:
Withdrawal should be negative and it should have red color: 
![image](https://user-images.githubusercontent.com/51168865/217810706-fe257214-ac25-43ec-931e-c378715b62d4.png)
The same applies to revenue share.